docs/en/index.md

```diff
@@ -71,29 +88,25 @@ class SendNewslettersCommand extends Command
 }
 ```
 
-The configure method is to name the command and specify arguments and options.
-They have a lot of options and you can read about them in Symfony Documentation.
 
-When the command is executed, the execute method is called with two arguments.
-First one is command input, which contains all the parsed arguments and options.
-The second one is command output which should be used to provide feedback to the developer which ran the command.
+In `__construct`, we setup the name of the command, then set some description and set all dependencies.
+  
+Every command contains an `execute` function, which is called by Symfony console, whenever the command should be executed. The arguments handle either input parameters, and/or allow you to write to output stream.
+ 
+Best practice is to return an exit code, which specifies if the command ran successfully or not. This code can be read by other applications, when they execute your app. This is useful for cronjobs. 
 
-Best practise is to return an exit code which specifies if the command ran successfully and can be read by other applications when executed.
+Every command needs to be registered in commands section of `config.neon`:
 
+```yml
+console:
+	commands:
+		- App\Console\SendNewslettersCommand
+```
 
 Extending
 ---------
 
-To add a command, simply register it as a service with tag `kdyby.console.command`
-
-```yml
-services:
-	newsletterCommand:
-		class: App\Console\SendNewslettersCommand
-		tags: [kdyby.console.command]
-```
-
-Alternatively you can use shorter syntax for registering command (without tag). It's useful when you have a lot of commands:
+To add a command, simply register it inside of the `console.commands` section:
```

Comment:
Maybe a new section or a specific note then? Will update the docs accoardingly.

Also, is this used and/or do you want to encourage this? Because if I'd write a Nette extension, that uses console, I'd include adding it to config.neon into my extension readme. IMHO update of config.neon should be best-practice of nextensions, shouldn't it?

My **personal** opinion is: do not use tags, but keep them for BC. Since all commands are also available as a service, you can inject them into your presenter/whatever and use them, as I did:

**BTW** -- do you want to add this to docs? Could add a section `Calling command from a command`, which would also explain the DI.

```php
class CleanupFocus extends Command
{
    /** @var Paths */
    private $pathsCommand;

    public function execute(InputInterface $input, OutputInterface $output)
    {
        // ...
        $frameInput = new ArgvInput(['actionname', '--all'], $this->pathsCommand->getDefinition());
        $this->pathsCommand->execute($frameInput, $output);
        // ...
    }
}
```